test/mpileup/depth.reg

```diff
@@ -60,6 +60,10 @@ P d5_b3aa.out  $samtools depth -aa -b xx.bed3 xx#depth3.bam
 # Fail as -a shouldn't output xn:16-20 (it doesn't with -r)
 F d5_b3a.out   $samtools depth -a  -b xx.bed3 xx#depth3.bam
 
+# All reference positions in a file with no alignments
+INIT x $samtools view -H -b -o noreads.bam xx#depth1.sam
+P d5_all4.out $samtools depth -aa noreads.bam
```

Comment:
noreads.bam should be added to `.gitignore`.

Reply:
Even better, if you call it e.g. `xx#depth-empty.bam` (or anything matching either `xx#depth*.bam*` or `*.tmp`) it'll already be ignored and also removed by `make testclean`.